src/SQLite3-Core-Tests/SQLite3BaseConnectionTest.class.st

```diff
@@ -344,6 +344,19 @@ SQLite3BaseConnectionTest >> testBooleanWriteThenRead [
 		]
 ]
 
+{ #category : #'tests - connections' }
+SQLite3BaseConnectionTest >> testCloseFile [
+
+	| fileName newConnection path |
+	path := Smalltalk os isWindows ifTrue: [ (Smalltalk os environment at: 'TMP'), '\' ] ifFalse: [ '/tmp/' ].
+	fileName := path, UUID new asString.
```

Comment:
can we extract into a new method #newUniqueFilename
(might be useful for other file based tests too)

Reply:
Done in the test class for now. I'll look at moving this to OSPlatform for Pharo 12.